src/Textual/encodings/textpreprocessing.jl

```diff
@@ -0,0 +1,124 @@
+"""
+    TextEncoding() <: Encoding
+
+Encodes `Paragraph`s by applying various textual transforms.
+
+
+Encodes
+- `Paragraph` -> `Paragraph`
+
+"""
+struct Sanitize <: Encoding
+    tfms
```

Comment:
Is `tfms` always a Vector? This can probably be constrained further.

Reply:
Hmm.. Not always a vector and it can be empty too. But if one transform is used that'll probably be used along with a couple of other transforms like remove_puncts, stripping case etc.

Reply:
What can it be other than a vector? Can you enumerate the possible types?

Reply:
I mean it can either be empty (empty vector?) or a vector. For example, in the case of text generation, we don't have any sanitization for the input data and here sanitization step can be skipped. But yes! it'll always be a vector then.